### PR TITLE
feature: ctrl h triggers helpscreen

### PR DESCRIPTION
**Pull request by coderofsalvation**
image

### CODE REVIEW

**Review by caseyjoy — COMMENTED** [review]
- Changing the manual to call Milkytracker "an all-in-one Tracker musicstudio" could be misleading, and makes it sound like it's a DAW
- Characters not in the Milkytracker bitmap fonts seem to be rendering as random other characters, as can be seen in the screenshot
    - The one in the screenshot is "…" being used instead of "...", but I suspect there's probably other characters as well
- Hex-encoding a large chunk of text is not ideal, it should be possible to put the text in a file as-is, or load it from a text or html file instead
- Using lynx + *nix utilities to convert the html might not be the best way to do this
- The optional Ok/Cancel setting is probably unnecessary for this dialog, there's nothing editable in it to cancel
- It might be nice to change the notice at the top of the new files to reflect that it's a new file based on another existing file, instead of one of the older original files
- It might be nice to have some sort of search or keep the links somehow, finding things in it might be difficult otherwise

**Comment by caseyjoy** [comment]
Actually, thinking about it a bit more, it might be easier to find things if the information got split up into multiple tabs, something like
image

**Comment by coderofsalvation** [comment]
Those are cool suggestions, but the amount of work involved sounds significant.
Before we zoom into your suggestions, can you give us a timeframe for all this?
Otherwise I think this little PR is a huge step forward for the end-user, since there is zero help in the application.
I would love to see the PR you're describing, but given the current state and activity of milkytracker development, i think we should not prioritize fictional features over small (working) PR's like this.
I agree with the text can be optimized a bit, but it's not a musthave, but a nicetohave to me.

File: src/tracker/DialogHelp.cpp [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+#include "PPUI.h"
+
+DialogHelp::DialogHelp(PPScreen* screen, 
+							 DialogResponder* responder,
+							 pp_int32 id,
+							 const PPString& caption,
+							 bool okCancel/* = false*/) :
+	PPDialogBase()
+{
+  char line[HELP_MAX_LINE];
```
Comment (peterbart-lmi) [inline src/tracker/DialogHelp.cpp:44]: Please adjust formatting.

File: src/tracker/DialogHelp.cpp [hunk 2]
```diff
@@ -0,0 +1,76 @@
+/*
+ *  tracker/DialogHelp.cpp
+ *
+ *  Copyright 2009 Peter Barth
```
[the remark this answers is not in the document]
Comment (coderofsalvation) [reply src/tracker/DialogHelp.cpp]: ok updated in latest commit

File: src/tracker/DialogHelp.cpp (hunk 1 above)
Comment (coderofsalvation) [reply src/tracker/DialogHelp.cpp]: ok updated in latest commit

**Review by peterbart-lmi — APPROVED** [review]

**Pull request merged**